transports/astartehttpendpoint.cpp

```diff
@@ -164,8 +164,7 @@ void HTTPEndpointPrivate::ensureCredentialsSecret()
     provider->setSslConfiguration(sslConfiguration);
     credentialsSecretProvider = provider;
 
-    QObject::connect(credentialsSecretProvider, &CredentialsSecretProvider::credentialsSecretReady, q, [this, q] (const QByteArray &credentialsSecret) {
-        qCDebug(astarteHttpEndpointDC) << "Credentials secret is: " << credentialsSecret;
+    QObject::connect(credentialsSecretProvider, &CredentialsSecretProvider::credentialsSecretReady, q, [this, q] () {
         connectToEndpoint();
```

Comment:
Could it be useful to keep the credentials secret debug log?

Reply:
mmm, we should avoid to log sensitive data.